### ISSUE TITLE
[FEATURE REQUEST]

### ISSUE THREAD

**Issue by kevintran128** [huggingface/autotrain-advanced#473]
### Feature Request

Autotrain with hyperparameters (epochs, learning rate, etc) fine-tuning and base model selection 

### Motivation

The value proposition of autotrain is to have a model automatically trained without much manual fine tuning or manual model selection. The Autotrain product a few years back had this feature. Could we bring it back in 2024?

**Comment by abhishekkrthakur** [comment]
yes, hopefully!
